### PR TITLE
Update erl-dns (use maps for named index)

### PR DESCRIPTION
**Pull request by urbanserj**

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {deps, [
     {lager, {git, "https://github.com/basho/lager", {tag, "3.2.4"}}},
     {jsx, {git, "https://github.com/talentdeficit/jsx.git", {tag, "master"}}},
-    {erldns, {git, "https://github.com/dcos/erl-dns.git", "deprecate-records_by_type"}},
+    {erldns, {git, "https://github.com/dcos/erl-dns.git", master}},
```
Comment (GoelDeepak) [inline rebar.config]: Should this follow the recommended style?  {branch, "master"}

**Review by GoelDeepak — CHANGES_REQUESTED** [review]

File: rebar.config (hunk 1 above)
Comment (urbanserj) [inline rebar.config]: Is there recommended style for rebar.config? I tried to use the style from this file. Nobody used `{branch, "master"}` here. Should I change it?

Comment (GoelDeepak) [inline rebar.config]: yeah, that is a recommendation from rebar.config. We try to change is as and when we update a particular reference. So, yeah it is not complete

**Review by GoelDeepak — APPROVED** [review]

**Comment by GoelDeepak** [comment]
LGTM

**Pull request merged**